package.json

```diff
@@ -165,7 +165,7 @@
 		"react-dom": "16.9.0",
 		"react-native": "0.61.5",
 		"react-test-renderer": "16.9.0",
-		"rimraf": "2.6.2",
+		"rimraf": "3.0.2",
```

Comment:
Changelog: https://github.com/isaacs/rimraf/blob/master/CHANGELOG.md

From my end, I confirm the major bump shouldn't affect our use 👍

Reply:
Most of tools drop support for no longer supported versions on Node these days. That’s usually the reason they do major version upgrades. 

Thanks for double checking 👍

I aligned versions of two dependencies  to avoid local copies in WP packages.